### PR TITLE
component unit test for AlertBox

### PR DESCRIPTION
**Pull request by aekong**

### CODE REVIEW

**Comment by akegan** [comment]
@aekong e2e tests are failing because of a different issue, so no need to worry! I can re-run once we fix the other issue.

**Comment by aekong** [comment]
> @aekong e2e tests are failing because of a different issue, so no need to worry! I can re-run once we fix the other issue.

ah cool yeah saw it in the circleci pipelines.  thanks for the heads up 👍

File: spec/javascript/components/molecules/AlertBox.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import AlertBox from '~/components/molecules/AlertBox'
+
+describe('AlertBox', () => {
+  test('it renders an alert box with a message', () => {
```
Comment (akegan) [inline spec/javascript/components/molecules/AlertBox.test.js:9]: one small nit is that I might separate these out into 2 separate tests, so one that is 'it renders an alert box with a message', and one that is 'it calls onClose click when the close button is clicked' or something like that

**Review by akegan — APPROVED** [review]
This looks great!

File: spec/javascript/components/molecules/AlertBox.test.js (hunk 1 above)
Comment (aekong) [reply spec/javascript/components/molecules/AlertBox.test.js:9]: that makes sense! i'll separate them out 👍

**Review by akegan — APPROVED** [review]
looks great! merge away once tests pass 🚀  thank you for these tests!!

**Pull request merged**